MyApplication3/app/src/main/java/com/example/myapplication/MatchData.java

```diff
@@ -3,188 +3,163 @@
 public class MatchData {
 
     //Test Data
-    boolean tf = false;
+    boolean testBool = false;

```

Comment:
What does testBool do here?

Reply:
I'm currently using it as a test for passing data between fragments and to make sure that it does that I think it should do without potentially making something that would mess with data that we're actually looking for